### PR TITLE
Replace GET requests with POST method

### PR DESCRIPTION
**Pull request by anitachen-appier**
### Purpose

Replace the GET http requests with POST methods when sending events and visitor data to the backend.
This change applies to both event tracking and visitor identification.
Note that we set the `connectTimeout` to 5 seconds and `readTimeout` to 10 seconds

### Result

All the data should be sent to the backend as before.

### Verification

Make sure you test both `push()` and `trackEvent()`. For example, 

``` kotlin
// Kotlin
val tracker = Woopra.getInstance(application).getTracker("<yourDomain>")
tracker.setVisitorProperty("name", "<yourName>")
tracker.setVisitorProperty("property1", "value1")
tracker.push()

tracker.setVisitorProperty("property2", "value2")
val event = WoopraEvent("event")
event.setProperty("key", "value")
tracker.trackEvent(event)
```

### CODE REVIEW

File: woopra/src/main/java/com/woopra/tracking/android/Utils.java [hunk 1]
```diff
@@ -8,21 +8,6 @@
...
-     */
-    public static String encode(String param) {
-        try {
-            return URLEncoder.encode(param, "utf-8");
-        } catch (Exception e) {
-            // will not throw an exception since utf-8 is supported.
-        }
-        return param;
-    }
-
```
Comment (anitachen-appier) [inline woopra/src/main/java/com/woopra/tracking/android/Utils.java:25]: No longer in use after changing to POST method.

**Comment by wubai-appier** [comment]
nits: We have to modify Constants.kt and modify the schema of endpoints from `http` to `https`.

**Review by wubai-appier — APPROVED** [review]
Approved with behavior review.

**Review by Derek-Chiu — APPROVED** [review]
Approved with the behavior review

**Pull request merged**